### DISCUSSION TITLE
question : how to get body data from HttpRequest object in Post Request

### DISCUSSION THREAD

**Discussion by brokensword2018** [chenshuo/muduo#594] [General]
it seems there is no way to get body data from HttpRequest object in post request;

**Comment by chenshuo** [comment]
Muduo http is not a full-featured http library, post request is not supported.
You may want to integrate some http server library with muduo if you need such features. Like libmicrohttpd or Mongoose.

**Comment by brokensword2018** [comment]
> Muduo http is not a full-featured http library, post request is not supported.
> You may want to integrate some http server library with muduo if you need such features. Like libmicrohttpd or Mongoose.

thank

> Muduo http is not a full-featured http library, post request is not supported.
> You may want to integrate some http server library with muduo if you need such features. Like libmicrohttpd or Mongoose.

thanks a lot.
